docker_images/sklearn/app/pipelines/tabular_regression.py

```diff
@@ -0,0 +1,6 @@
+from app.pipelines.tabular_classification import TabularClassificationPipeline
```

Comment:
Could you also add a template in https://github.com/huggingface/api-inference-community/tree/main/docker_images/common/app/pipelines please? This can be the same as in https://github.com/huggingface/api-inference-community/blob/main/docker_images/common/app/pipelines/tabular_data_pipeline.py